### ISSUE TITLE
"General download options..." window won't open

### ISSUE THREAD

**Issue by starpotion** [axcore/tartube#394]
### What operating system are you using?
Windows 11

### What version of Tartube are you using?
v2.3.484

### What happens when you run Tartube from a terminal window?
Tartube seems to run fine when running from the terminal, but the window still doesn't appear when clicking the button

I'm trying to figure out cookies and modify my settings so I can download age restricted videos. I originally entered my username and password into the general download options which I believe may have something to do with it.

**Comment by axcore** [comment]
Run Tartube from the mingw64 terminal, as described [here](https://tartube.sourceforge.io/#tartube-won-t-install-won-t-run-doesn-t-work). When the options window fails to open, a load of python errors should appear in the terminal, and then you can show me those errors.

**Comment by starpotion** [comment]
Sure thing, here's what I got:

```
rekyuu@REKYUU-PC MINGW64 ~
$ cd /home/user/tartube

rekyuu@REKYUU-PC MINGW64 /home/user/tartube
$ python3 tartube/tartube

(tartube:8088): GLib-GIO-WARNING **: 13:40:56.660: Unexpectedly, UWP app `GravitGmbH.GravitDesigner_3.5.73.0_x64__78ng2gh36rjtw' (AUMId `GravitGmbH.GravitDesigner_78ng2gh36rjtw!GravitDesigner') supports 1 extensions but has no verbs
C:/Users/rekyuu/AppData/Local/Tartube/msys64/mingw64/lib/python3.9/site-packages/gi/overrides/Gio.py:42: Warning: unable to set property 'strikethrough-set' of type 'gboolean' from value of type 'gchararray'
  return Gio.Application.run(self, *args, **kwargs)
C:/Users/rekyuu/AppData/Local/Tartube/msys64/mingw64/lib/python3.9/site-packages/gi/overrides/Gio.py:42: Warning: unable to set property 'underline-set' of type 'gboolean' from value of type 'gchararray'
  return Gio.Application.run(self, *args, **kwargs)
C:/Users/rekyuu/AppData/Local/Tartube/msys64/mingw64/lib/python3.9/site-packages/gi/overrides/Gio.py:42: Warning: unable to set property 'weight-set' of type 'gboolean' from value of type 'gchararray'
  return Gio.Application.run(self, *args, **kwargs)
C:/Users/rekyuu/AppData/Local/Tartube/msys64/mingw64/lib/python3.9/site-packages/gi/overrides/Gio.py:42: Warning: unable to set property 'style-set' of type 'gboolean' from value of type 'gchararray'
  return Gio.Application.run(self, *args, **kwargs)
Traceback (most recent call last):
  File "C:/Users\rekyuu\AppData\Local\Tartube\msys64\home\user\tartube\tartube/mainapp.py", line 22345, in on_menu_general_options
    config.OptionsEditWin(self, self.general_options_obj)
  File "C:/Users\rekyuu\AppData\Local\Tartube\msys64\home\user\tartube\tartube/config.py", line 4092, in __init__
    self.setup()
  File "C:/Users\rekyuu\AppData\Local\Tartube\msys64\home\user\tartube\tartube/config.py", line 109, in setup
    self.setup_tabs()
  File "C:/Users\rekyuu\AppData\Local\Tartube\msys64\home\user\tartube\tartube/config.py", line 4346, in setup_tabs
    self.setup_ytdlp_tab()
  File "C:/Users\rekyuu\AppData\Local\Tartube\msys64\home\user\tartube\tartube/config.py", line 6142, in setup_ytdlp_tab
    self.setup_ytdlp_cookies_tab(inner_notebook)
  File "C:/Users\rekyuu\AppData\Local\Tartube\msys64\home\user\tartube\tartube/config.py", line 6780, in setup_ytdlp_cookies_tab
    keyring = keyring[1:]
TypeError: 'NoneType' object is not subscriptable
```

**Comment by axcore** [comment]
Fixed in v2.3.518. The new code is only available from github/pypi at the moment. If you want to update now, you can just overwrite the source code in your installation folder, as described [here](https://github.com/axcore/tartube#71-tartube-wont-installwont-rundoesnt-work).
